ports/vcpkg-tool-meson/portfile.cmake

```
@@ -5,17 +5,17 @@
 set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
 
 set(program MESON)
-set(program_version 0.62.1)
+set(program_version 0.62.2)
 set(program_name meson)
 set(search_names meson meson.py)
 set(interpreter PYTHON3)
 set(apt_package_name "meson")
 set(brew_package_name "meson")
-set(ref bb91cea0d66d8d036063dedec1f194d663399cdf)
+set(ref 0.62.2)
```

```suggestion
set(ref 373d18bc4dfca088aaa92a2d23ed486f3faf527f)
```
since tags can be changed by upstream.